common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -105,17 +105,30 @@ $sequence--border-color: #C8C8C8;
         display: block;
         padding: 0;
         position: relative;
+        text-align: center;
 
-        &:hover, &:focus {
-          background-color: $white;
-          background-repeat: no-repeat;
-          background-position: center 14px;
+        .icon {
+          line-height: 42px;
+          font-size: 90%;
+          color: rgb(80, 80, 80);
+        }
+
+        &.inactive {
+
+          .icon {
+            // color: rgb(160, 160, 160); # Lighter gray, but not sure why?
+            color: rgb(80, 80, 80);
+          }
         }
 
         &.active {
           @extend %ui-depth1;
           background-color: $white;
 
+          .icon {
+            color: rgb(182,37,103);
```

Comment:
Also, I have some visual feedback on this - I may make a commit on this branch to address it. I'll ping you on it if I do.